eox_core/api/data/v1/viewsets.py

```diff
@@ -35,7 +35,7 @@ class DataApiViewSet(mixins.ListModelMixin,
     """
     A generic viewset for all the instances of the data-api
     """
-    authentication_classes = (OAuth2Authentication, SessionAuthentication)
+    authentication_classes = (get_bearer_authentication(), SessionAuthentication)
```

Comment:
Create a constant with the result in order to use it in new classes

Reply:
do you mean the same solution of views.py file?